### PR TITLE
added context and reducer

### PR DESCRIPTION
**Pull request by aliamaanansari**

### CODE REVIEW

File: src/store/data.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
+{
```
Comment (poojakelkar) [inline src/store/data.json:1]: even tough you'll fetch this data from api later on, please add proper questions and remove this dummy questions and answers

**Review by poojakelkar — CHANGES_REQUESTED** [review]

File: src/store/reducer.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { types } from "./constants";
+import data from "./data.json";
+
+export const reducer = (currentState, action) => {
+  console.log({ ...currentState }, "previous");
```
Comment (poojakelkar) [inline src/store/reducer.js]: please remove unwanted console logs

File: src/store/data.json (hunk 1 above)
Comment (aliamaanansari) [reply src/store/data.json:1]: have removed this file as i've integrated api locally, will create a PR soon

File: src/store/reducer.js (hunk 2 above)
Comment (aliamaanansari) [reply src/store/reducer.js]: thanks, removed them

**Pull request merged**